packages/grafana-prometheus/src/components/PromCheatSheet.tsx

```diff
@@ -29,23 +31,44 @@ const CHEAT_SHEET_ITEMS = [
   },
 ];
 
-export const PromCheatSheet = (props: QueryEditorHelpProps<PromQuery>) => (
-  <div>
-    <h2>PromQL Cheat Sheet</h2>
-    {CHEAT_SHEET_ITEMS.map((item, index) => (
-      <div className="cheat-sheet-item" key={index}>
-        <div className="cheat-sheet-item__title">{item.title}</div>
-        {item.expression ? (
-          <button
-            type="button"
-            className="cheat-sheet-item__example"
-            onClick={(e) => props.onClickExample({ refId: 'A', expr: item.expression })}
-          >
-            <code>{item.expression}</code>
-          </button>
-        ) : null}
-        <div className="cheat-sheet-item__label">{item.label}</div>
-      </div>
-    ))}
-  </div>
-);
+export const PromCheatSheet = (props: QueryEditorHelpProps<PromQuery>) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div>
+      <h2>PromQL Cheat Sheet</h2>
+      {CHEAT_SHEET_ITEMS.map((item, index) => (
+        <div className={styles.cheatSheetItem} key={index}>
+          <div className={styles.cheatSheetItemTitle}>{item.title}</div>
+          {item.expression ? (
+            <button
+              type="button"
+              className={styles.cheatSheetExample}
+              onClick={(e) => props.onClickExample({ refId: 'A', expr: item.expression })}
+            >
+              <code>{item.expression}</code>
+            </button>
+          ) : null}
+          {item.label}
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  cheatSheetItem: css({
```

Comment:
I am seeing a lot of cheat sheets. Should these be standardized? Or at least share styles to some extent?

Reply:
probably, but i think that's then up to the datasources tbh
there's also a question of where that component would live, cause it shouldn't be `@grafana/ui` 😅